packages/testkit-backend/src/request-handlers.js

```diff
@@ -327,8 +326,9 @@ export function SessionLastBookmarks (_, context, data, wire) {
 }
 
 export function SessionWriteTransaction (_, context, data, wire) {
-  const { sessionId, txMeta: metadata } = data
+  const { sessionId } = data
   const session = context.getSession(sessionId)
+  const metadata = context.binder.objectToNative(data.txMeta)
```

Comment:
You probably wan't to add the same treatment to the handlers in the `request-handlers-rx.`